tensorflow/compiler/mlir/python/mlir.h

```diff
@@ -44,6 +45,15 @@ std::string ImportFunction(const std::string &functiondef_proto,
                            bool show_debug_info, TFE_Context *context,
                            TF_Status *status);
 
+// This wrapper passes the graph_def taking names of input nodes, its types
+// shape and the ouput nodes as parameters to MLIR.
```

Comment:
its types shape <-- please expand and rephrase this for clarify. `... the shapes and types of its inputs`.  In general, expand and improve this doc comment for clarity.

Reply:
Done.